Resources/Prototypes/Entities/Structures/Doors/Turnstiles/turnstiles.yml

```diff
@@ -0,0 +1,126 @@
+﻿- type: entity
+  id: Turnstile
+  parent: BaseStructure
+  name: turnstile
+  description: One-way access - control the flow of foot traffic.
+  components:
+  - type: MeleeSound
+    soundGroups:
+      Brute:
+        path:
+          "/Audio/Weapons/smash.ogg"
+  - type: InteractionOutline
+  - type: Sprite
+    sprite: Structures/Doors/turnstile.rsi
+    noRot: true
+    drawdepth: Mobs
+    state: idle
+  - type: Rotatable
+  - type: Transform
+    noRot: false
+    snapCardinals: true
+  - type: Fixtures
+    fixtures: # tall turnstile has two thin walls on each side.
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.49,-0.49,-0.44,0.49"
+        density: 200
+        mask:
+        - FullTileMask
+        layer:
+        - AirlockLayer
+      fix2:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "0.44,-0.49,0.49,0.49"
+        density: 200
+        mask:
+        - FullTileMask
+        layer:
+        - AirlockLayer
+  - type: Turnstile
+    turnSound:
+      path: /Audio/Effects/unbuckle.ogg
+    bumpSound:
+      collection: FenceRattle
+    spinnerPrototype: TurnstileSpinner
+    spinnerAnchorPoint: "0.5,0.0"
+  - type: Appearance
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: FlimsyMetallic
+  - type: Destructible
+    thresholds:
+    - trigger:
```

Comment:
like sloth said add a 500 or whatever trigger that uses delete behaviour

Reply:
I can't seem to find that part of the discussion - do you have a link so I can see?